### ISSUE TITLE
Extension suddenly not working since yesterday

### ISSUE THREAD

**Issue by wombat2024** [sixem/tiktok-download-buttons#15]
Tried both firefox and chrome, no luck... please check when available, million thanks! 🙏

**Comment by wombat2024** [comment]
Also may be interested in hd version downloading option in the future like
https://tikdownloader.io/en

**Comment by sixem** [comment]
Hmm, are the buttons gone as well? Because it's working just fine for me, the buttons and the actual downloading, that is. It could be a regional issue, where they're rolling out changes for certain regions before others, but I don't have a good way of checking for this at the moment.

> Also may be interested in hd version downloading option in the future like https://tikdownloader.io/en

A lot of these HD sites either just upscales the videos or use an actual custom API key (at least the last time I checked), and this addon uses the open TikTok API to get the video URLs, and no 3rd party sites.

I did have a quick check, and I do think there are better quality versions available that I could possibly download instead, and I can have a look at this when I have time. I think I originally avoided this because some of these higher quality versions used a different encoding which weren't always playable directly in the browser, or at least for Firefox, but I'll have a look again.
